docs/faq.md

```diff
@@ -21,7 +21,7 @@ the documentation sources.
 
 ### How much does Engine cost?
 
-Docker Engine is 100% free. It is open source, so you can use it without paying.
+The open source Docker Engine is available for free. Commercial support is also available for the Docker Engine as a standalone product or as part of the Docker Datacenter subscription. Pricing details are available at [https://www.docker.com/pricing](www.docker.com/pricing)
```

Comment:
There are separate issues here. The core open-source project itself should not have references to commercial products (unless it's wide open for anyone to add a ref to their $ offering) and when people download the src they should just get ad-free stuff. How docker.com displays this is separate issue. There should be an ad-free oss docker website and then a commercial dockerInc website. The fact they're merged today is an issue that is being looked at - but in the meantime we don't need to make it more mixed-up.